c7n/mu.py

```diff
@@ -991,6 +991,67 @@ def remove(self, func):
                     raise
 
 
+class SimpleNotificationServiceSubscription(object):
+    """ Subscribe a lambda to one or more SNS queues
+    """
+
+    iam_delay = 1.5
+
+    def __init__(self, session_factory, log_groups, filter_pattern):
+        self.log_groups = log_groups
+        self.filter_pattern = filter_pattern
+        self.session_factory = session_factory
+        self.session = session_factory()
+        self.client = self.session.client('logs')
+
+    def add(self, func):
+        lambda_client = self.session.client('lambda')
+        for group in self.log_groups:
+            log.info(
+                "Creating subscription filter for %s" % group['logGroupName'])
+            region = group['arn'].split(':', 4)[3]
```

Comment:
afaicr region is also available via self.session.region_name fwiw, this usage is fine. note we target sns and lambda have to be provisioned into same region.